src/app/components/track-controls/track-controls.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, ViewChild, HostListener, Input, EventEmitter, Output } from
 import { PlayerService, PlaybackStates } from '../../services/player.service';
 import { MatBottomSheet } from '@angular/material';
 import { SettingsComponent } from '../settings/settings.component';
+import { connectableObservableDescriptor } from 'rxjs/internal/observable/ConnectableObservable';
```

Comment:
Is this import used?

Reply:
Not by me. Wonder if it was something auto added by VSC when I was messing with stuff. Removed.